.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 18.9b0
```

Comment:
I'm guessing this is just to align with the other repos but why are we using a pre-release version of black? Does pre-commit work with the latest version (22.10.0)?

Reply:
Version of flask8 is not 5.0.4, version is black is 22.8.0 and pre-commit is now version 2.20.0. These are all the latest versions.